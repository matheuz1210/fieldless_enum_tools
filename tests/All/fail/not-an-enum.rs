use fieldless_enum_tools::All;

#[derive(All)]
struct NotAnEnum {
    field1: u8,
    field2: u8,
}

fn main() {}
