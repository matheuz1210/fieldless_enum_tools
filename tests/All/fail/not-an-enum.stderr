error: #[derive(All)] expected an enum, yet a struct was provided
 --> tests/All/fail/not-an-enum.rs:4:1
  |
4 | struct NotAnEnum {
  | ^^^^^^
